src/components/actions-sheet.tsx

```diff
@@ -0,0 +1,163 @@
+import React, {useEffect} from 'react';
```

Comment:
how about to split logic into iOS and android files and in iOS use native realization?